README.md

```diff
@@ -139,8 +139,25 @@ julia> Box2{Int}(1) == Box2{Any}(1)
 false
 
 julia> hash(Box2{Int}(1))
-0xb7650cb555d6aafa
+0x467811eefea1d458
 
 julia> hash(Box2{Any}(1))
-0xefe691a94f296c61
+0x3042fd2f8fe839d7
 ```
+
+## Specifying the "type seed"
+
+When we compute the hash function, we start with a "seed" particular to the type being hashed.
+By default, the seed is computed as `hash(:TypeName)` if `typearg=false` (which is the default),
+where `hash` is the hash function specified with `hashfn=`, or `Base.hash` if `hashfn=` was not specified.
+If `typearg=true` was specified, then the seed is computed as `hash(Type)`,
+where `Type` is the type of the instance, including any type arguments.
```

Comment:
is this still true? it's a bit more nuanced now I think

Reply:
No, this needs to be rewritten.